pkg/operator/operator.go

```diff
@@ -405,6 +405,34 @@ func New(
 	}
 	o.controllersToRunFunc = append(o.controllersToRunFunc, csrController.Run, o.ruleController.Run, o.relabelController.Run)
 
+	csrFederateController, err := csr.NewClientCertificateController(
```

Comment:
It might be worth commenting about the purpose of this controller? e.g.

```suggestion
	// csrFederateController runs a controller that requests a client TLS certificate for the telemeter client. This certificate is used to authenticate against the Prometheus /federate API endpoint.
	csrFederateController, err := csr.NewClientCertificateController(
```